saleor/graphql/order/mutations/order_update_shipping.py

```diff
@@ -137,5 +145,13 @@ def perform_mutation(  # type: ignore[override]
 
         order.save(update_fields=SHIPPING_METHOD_UPDATE_FIELDS)
         # Post-process the results
-        call_order_event(manager, WebhookEventAsyncType.ORDER_UPDATED, order)
+
+        event_to_emit = (
+            WebhookEventAsyncType.DRAFT_ORDER_UPDATED
```

Comment:
You should also emit the event for the `return` from the line 122

Reply:
as discussed - it was out of the initial scope, but added + incldued in changelog as a separate change